sdcm/cluster_gce.py

```diff
@@ -366,14 +373,17 @@ def _create_node_with_retries(self,
                                   create_node_params: dict[str, Any],
                                   spot: bool = False) -> GCENode:
         try:
-            return self._gce_services[dc_idx].create_node(**create_node_params)
+            response = self._gce_services[dc_idx].create_node(**create_node_params)
+            self.log.info(f"Got GCE create node response:\n{response}")
```

Comment:
please put this print in debug

Reply:
I'll remove this, as it's just a debug log for my needs

Reply:
Fixed (removed)